OpenZaak_Mission_Vision_Scope.md

```diff
@@ -0,0 +1,46 @@
+# Mission statement
+What: purpose of OpenZaak
+
+OpenZaak wants to help Dutch municipalities and other government agencies to implement Zaakgericht Werken (ZGW, a Dutch form of case management for municipalities). Moreover it focuses on the registries that are necessary to store the data that is created and needed for ZGW.
+
+Why OpenZaak
+
+We believe that Zaakgericht Werken is a best practice for handling requests from citizens and companies. As such ZGW is a way of working that helps to efficiently organise the backoffice of municipalities on the one hand and to record data in a standardised way that is compliant with laws and regulations.
```

Comment:
Als je 'on the one hand' gebruikt, moet je dan ook 'on the other hand' gebruiken?